src/unity/python/turicreate/test/test_one_shot_object_detector.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+import unittest
+import pytest
+from . import util as test_util
+import turicreate as tc
+import tempfile
+import numpy as np
+import platform
+import sys
+import os
+from turicreate.toolkits._main import ToolkitError as _ToolkitError
+from turicreate.toolkits._internal_utils import _raise_error_if_not_sarray, _mac_ver
+import coremltools
+
+_CLASSES = ['logo_a', 'logo_b', 'logo_c', 'logo_d']
+
+def _get_data(feature, target):
+    from PIL import Image as _PIL_Image
+    rs = np.random.RandomState(1234)
+    def from_pil_image(pil_img, image_format='png'):
+        if image_format == 'raw':
+            image = np.array(pil_img)
+            FORMAT_RAW = 2
+            return tc.Image(_image_data=image.tobytes(),
+                            _width=image.shape[1],
+                            _height=image.shape[0],
+                            _channels=image.shape[2],
+                            _format_enum=FORMAT_RAW,
+                            _image_data_size=image.size)
+        else:
+            with tempfile.NamedTemporaryFile(mode='w+b', suffix='.' + image_format) as f:
+                pil_img.save(f, format=image_format)
+                return tc.Image(f.name)
+
+    num_examples = 100
+    num_starter_images = 5
+    max_num_boxes_per_image = 10
+    classes = _CLASSES
+    images = []
+    FORMATS = ['png', 'jpeg', 'raw']
+    for i in range(num_examples):
```

Comment:
nit: in Python when a variable is never intended to actually be used it's standard to name the variable `_`.